kettle-plugins/spark/src/main/java/org/pentaho/di/job/entries/spark/JobEntrySparkSubmit.java

```diff
@@ -76,7 +76,7 @@
     name = "JobEntrySparkSubmit.Title", description = "JobEntrySparkSubmit.Description",
     categoryDescription = "i18n:org.pentaho.di.job:JobCategory.Category.BigData",
     i18nPackageName = "org.pentaho.di.job.entries.spark",
-    documentationUrl = "http://wiki.pentaho.com/display/EAI/Spark+Submit" )
+    documentationUrl = "JobEntrySparkSubmit.DocumentationUrl" )
```

Comment:
Just like the other pull request, we just need the URL fragment here, PDI will handle the assembly of the full URL with the version attached to me.